Library/Styles/LoginStyles.swift

```diff
@@ -100,6 +100,17 @@ public let loginRootStackViewStyle =
         : .init(topBottom: Styles.grid(2), leftRight: Styles.grid(3))
 }
 
+public let facebookConfirmationStackViewStyle =
+  UIStackView.lens.layoutMarginsRelativeArrangement .~ true
+    <> UIStackView.lens.layoutMargins %~~ { _, stack in
+      stack.traitCollection.isRegularRegular
+        ? .init(topBottom: Styles.grid(10), leftRight: Styles.grid(20))
+        : .init(
+          topBottom: stack.traitCollection.isVerticallyCompact ? Styles.grid(2) : Styles.grid(10),
+          leftRight: Styles.grid(3)
```

Comment:
@theginbin tank you for reviewing this! This was my original thought and fix - directly in loginRootStackViewStyle to be consistent between all login views . But then we discussed it with @justinswart and decided to do it only for facebookConfirmation because we thought that we may want different margins for different login views. 

Based on @stephencelis  suggestion (above) at the moment I implemented it as an expansion on loginRootStackViewStyle like this:
```swift
public let facebookConfirmationStackViewStyle = loginRootStackViewStyle
    <> UIStackView.lens.layoutMargins..UIEdgeInsets.lens.topBottom %~~ { (topBottom, stack)  in
      return (stack.traitCollection.isVerticallyCompact ? Styles.grid(2) : topBottom.0, topBottom.1)
}    
```
But if our intension is to be consistent between all login views I probably should go back to my original fix..
Should we have a quick discussion together with @justinswart, and then I will do the fix base on what we decide?